runner/src/server/plugins/engine/pageControllers/RepeatingSummaryPageController.ts

```diff
@@ -142,7 +150,9 @@ export class RepeatingSummaryPageController extends PageController {
         actions: {
           items: [
             {
-              href: `?removeAtIndex=${i}${view ? `&view=${view}` : ``}`,
+              href: `?form_session_identifier=efg-895656&removeAtIndex=${i}${
+                view ? `&view=${view}` : ``
+              }${form_session_identifier}`,
```

Comment:
I don't quite get this functionality, why is `form_session_identifier=efg-895656` hardcoded? but then we add `form_session_identifier` on at the end? 

With my other comment, I don't think the query param is being passed into this method, but rather the raw identifier, so wouldn't we need to either:

A. Add `&form_session_identifier=` before `${form_session_identifier}` in the current method, and remove the hardcoded `form_session_identifier=efg-895656`
B. Pass the `form_session_identifier` variable into the method instead of `response.query.form_session_identifier`, but still remove the hardcoded `form_session_identifier=efg-895656`

Reply:
Yep that was left over test data. Eagerness to get it out made me miss this. Passing through form session identifier

Reply:
👍 nice thanks for fixing, looks good.